src/svelte/Components/Modals/MacroModal/RunInfo.svelte

```diff
@@ -0,0 +1,29 @@
+<div class="dropdown dropdown-end dropdown-right">
+  <label tabindex="0" class="btn btn-circle btn-ghost btn-xs text-info">
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      class="w-4 h-4 stroke-current"
+      ><path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        stroke-width="2"
+        d="M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
+      /></svg
+    >
+  </label>
+  <div
+    tabindex="0"
+    class="card compact dropdown-content shadow bg-base-100 rounded-box w-64"
+  >
+    <div class="card-body">
+      <h2 class="card-title">Ausführbare Befehle</h2>
+      <p>
+        Wenn du ein Makro als ausführbar kennzeichnest und einen Befehl mit
+        mehreren Zeilen zu verwenden, dann verwende bitte ein "^" am ende jeder
+        Zeile des Befehls.
```

Comment:
```suggestion
    Zur Ausführung der Befehle wird im Hintergrund die cmd.exe verwendet. Entsprechend müssen ausführbare Befehle die entsprechende Syntax verwenden. Vor einem Zeilenumbruch muss zum Beispiel die Zeile mit einem "^" beendet werden. Für weitere Infos siehe hier (https://ss64.com/nt/syntax-esc.html)
```

Irgendwie so vielleicht? Ist jedenfalls kein richtiger Satz, den du da geschrieben hast :D

Reply:
vielleicht kann man das auch einfach kürzer formulieren, z.B.:
Zur Ausführung der Befehle wird die cmd.exe verwendet, entsprechend müssen Befehle die richtige Syntax verwenden. Vor einem Zeilenumbruch muss bspw. die Zeile mit "^" beendet werden. Für weitere Infos: link (vielleicht könnte man das weitere Infos auch weglassen)